File: Face.java
public class Face{

  /** Entier permettant le swap entre deux valeur sans utiliser de tmp */
  private int color;
  /** String représentant une couleur */
  private String type;

  public Face(int color){
    this.color = color;
  }

  public int getColor(){
    return color;
  }

  public void setColor(int color){
    this.color = color;
  }

  public String getType(){
    return type;
  }

  public void setType(String type){
    this.type = type;
  }

  public String toString(){
    return ""+color+":"+type;
  }
}

File: Partie_LORENZELLI/Face.py
class Face:

    def __init__(self, color, type):
        self.__color = color
        self.__type = type

    def get_color(self):
        return self.__color

    def set_color(self, color):
        self.__color = color

    def get_type(self):
        return self.__type

    def set_type(self, type):
        self.__type = type
